CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## neptune-pytorch 0.1.0 (initial release)
+
+### Features
+- Create `NeptuneLogger` for logging metadata
```

Comment:
Maybe we could also add the link to the relevant PR?

Reply:
Done